src/Interpreters/HashJoin/HashJoin.cpp

```diff
@@ -1049,9 +1049,10 @@ DataTypePtr HashJoin::joinGetCheckAndGetReturnType(const DataTypes & data_types,
     if (right_table_keys.columns() != num_keys)
         throw Exception(
             ErrorCodes::NUMBER_OF_ARGUMENTS_DOESNT_MATCH,
-            "Number of arguments for function joinGet{} doesn't match: passed, should be equal to {}",
+            "Number of arguments for function joinGet{} doesn't match: passed {}, should be equal to {}",
```

Comment:
As per my general comment, this is a specialized error message checking if the number of join table key columns matches the supplied number of `join_keys`. However the way it is written implies that it is complaining about the overall number of arguments, so let's update it to be a bit more clear. I propose something like:

```suggestion
            "Number of join_keys does not match number of right table key columns for function joinGet{}: passed {}, should be {}",
```